src/LinearMap.jl

```diff
@@ -61,7 +61,7 @@ Return a linear map of a convex set by a scalar value.
 The linear map of the convex set.
 """
 function *(a::Real, S::LazySet)::LinearMap
-    return LinearMap(sparse(a*I, dim(S)), S)
+    return LinearMap(a * speye(typeof(a), dim(S)), S)
```

Comment:
how are left and right different? is this related to:

> fix a type conversion bug in LinearMap with a constant (was always a matrix of type Int)

Reply:
Yes, I separated this in a single commit. I can try again, but I think it had the `Int` type previously.

Reply:
Hm, you are right, I cannot reproduce the problem anymore...

Reply:
```julia
sparse(1*I, 5)
5×5 SparseMatrixCSC{Int64,Int64}
a=1.; a * speye(typeof(a), 5)
5×5 SparseMatrixCSC{Float64,Int64}
```
This is what I meant.